etest/profile.py

```diff
@@ -0,0 +1,91 @@
+"""Profile definitions and builders."""
+
+
+arm_versions = {
+    "armv5": "armv5tel",
+    "armv6": "armv6j_hardfp",
+    "armv7": "armv7a_hardfp",
+}
+
+
+class InvalidProfileError(Exception):
+    """The profile is invalid."""
+
+    pass
+
+
+class Profile:
+    """A profile."""
+
+    def __init__(self, quiet: bool, arch: str, libc: str, hardened: bool, multilib: bool, systemd: bool) -> None:
+        """Initialize parameters."""
+        self.quiet = quiet
+
+        self.arch = arch
+        self.docker_arch = arch
+        self.pkg_arch = arch
+
+        self.libc = libc
+        self.hardened = hardened
+
+        self.multilib = multilib
+
+        self.systemd = systemd
+
+    def sanitize(self) -> None:
```

Comment:
Is this sanitisation or canonicalisation?  It reads like the latter to me so let's rename the method.

Reply:
Ill change its name to "standarize", since "canonicalize" feels like a mouthful :p I hope its ok

Reply:
Ha, it's canonical to call this canonicalise but since it's internal only (should have a leading underscore) standardise seems fine to me.  Sanitize doesn't fit in my opinion though.